jmix-flowui/flowui/src/main/java/io/jmix/flowui/exception/ConstraintViolationExceptionHandler.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2022 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.flowui.exception;
+
+import io.jmix.core.MessageTools;
+import io.jmix.flowui.Notifications;
+import io.jmix.flowui.component.validation.ValidationErrors;
+import io.jmix.flowui.view.ViewValidation;
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.ConstraintViolationException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Set;
+
+@Component
+public class ConstraintViolationExceptionHandler extends AbstractUiExceptionHandler {
```

Comment:
Please add javadoc explaining when this class is used and what it does.

Reply:
Done